astropy/table/tests/test_groups.py

```diff
@@ -576,6 +576,18 @@ def np_add(x):
     assert tga.pformat() == [" a ", "---", "  0", "  1", "  2"]
 
 
+def test_table_aggregate_reduceat_empty():
+    for masked in (False, True):
+        tg = Table(
+            {
+                "action": np.asarray([], dtype=str),
+                "duration": np.asarray([], dtype=float),
+            }
+        )
+        tga = tg.group_by("action").groups.aggregate(np.sum)
+        assert tga.pformat() == ["action col1", "------ ----"]
```

Comment:
This test failed on `main` without this patch (I tested it). However, unlike the expected outcome stated by user, the second column name is "col1" instead of "duration". When you use non-empty arrays on `main`, it is still like this, so I am not sure if this is a different bug or what.

Reply:
Nvm it was my bad. Fixed.